### PR TITLE
docs(graphql): added warning

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: 
#1836

## What is the new behavior?
N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This PR, Added warning in case you use `apollo-server-fastify` package with GraphQL

### CODE REVIEW

File: content/graphql/quick-start.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ export class AppModule {}
 
 As mentioned, these options will be forwarded to the `ApolloServer` constructor.
 
+> warning **Warning** If you use `apollo-server-fastify` package, enable in the `GraphQLModule` configuration the `disableHealthCheck` setting to `true`.
```
Comment (kamilmysliwiec) [inline content/graphql/quick-start.md:61]: This happens only if you have multiple GraphQL endpoints in a single app.

Comment (Tony133) [reply content/graphql/quick-start.md]: Ok 👍, I made the fix

File: content/graphql/quick-start.md [hunk 2]
```diff
@@ -58,6 +58,8 @@ export class AppModule {}
 
 As mentioned, these options will be forwarded to the `ApolloServer` constructor.
 
+> warning **Warning** If you use the `apollo-server-fastify` package with multiple GraphQL endpoints in a single application, enable the `disableHealthCheck` setting to` true` in the configuration` GraphQLModule`.
+
```
Comment (kamilmysliwiec) [inline content/graphql/quick-start.md:62]: If this suggestion applies only to the case when a developer has multiple endpoints, "Getting started with GraphQL & TypeScript" is not the right place for this hint

Comment (Tony133) [reply content/graphql/quick-start.md:62]: I agree, I had put it there as there is the example of the `GraphQLModule` configuration but it is not very sure it was really good. 

If we move it to the section dedicated to `Multiple endpoints`?

Comment (kamilmysliwiec) [reply content/graphql/quick-start.md:62]: Sounds good

Comment (Tony133) [reply content/graphql/quick-start.md]: Ok, moved it in `Multiple endpoints`

File: content/graphql/quick-start.md [hunk 3]
```diff
@@ -80,6 +80,8 @@ GraphQLModule.forRoot({
 }),
 ```
 
+> warning **Warning** If you use the `apollo-server-fastify` package with multiple GraphQL endpoints in a single application, enable the `disableHealthCheck` setting to` true` in the configuration` GraphQLModule`.
```
Comment (kamilmysliwiec) [inline content/graphql/quick-start.md:83]:
```suggestion
> warning **Warning** If you use the `apollo-server-fastify` package with multiple GraphQL endpoints in a single application, make sure to enable the `disableHealthCheck` setting in the `GraphQLModule` configuration.
```

**Pull request merged**